### DISCUSSION TITLE
umi最新版，默认target ie11, build后``字符串没转化造成ie11解析umi.xxxxxx.js 文件报错

### DISCUSSION THREAD

**Discussion by vilisper** [umijs/umi#6358] [Q&A]
umi最新版，默认target ie11, build后``字符串没转化造成ie11解析umi.xxxxxx.js 文件报错， 页面就打不开了。

**Comment by xiaohuoni** [comment]
``字符串 在项目中还是在第三方依赖中？

**Comment by Map1en** [comment]
应该用了plugin-esbuild，然后plugin-esbuild的target有问题，已经修改，等发版。
